libraries/adaptive-expressions-ie11/webpack.config.js

```diff
@@ -33,14 +33,14 @@ module.exports = () => {
                         },
                     ],
                     include: [
-                        path.resolve(__dirname, './node_modules/adaptive-expressions'),
+                        path.resolve(__dirname, '../adaptive-expressions'),
```

Comment:
Does require.resolve work? That should abstract this all away. You would just pass the name of the package.

Reply:
Nope require.resolve doesnt work
`require.resolve('adaptive-expressions')`